### ISSUE TITLE
Clicking "Places" map icons doesn't do anything

### ISSUE THREAD

**Issue by sstruemph** [WordImpress/Google-Maps-Builder#212]
Steps to replicate:

(tested with free and pro)

1. Create new map
2. Enable Google Places: Google Places > Show Places? > select "Yes"
3. Choose a place category to show. For testing I chose "Bakeries". Click update map
4. Publish map
5. Click on an icon for a "Place"
6. Nothing happens but there are errors in the console

Errors:
![mapsbuilder-free-places-error](https://cloud.githubusercontent.com/assets/3521793/17344905/6a9252ec-58c9-11e6-9e83-a966bdb69dd5.jpg)

**Comment by DevinWalker** [comment]
@sstruemph just to make sure - are you using the latest version?

**Comment by sstruemph** [comment]
@DevinWalker Does this look right?

![gmb](https://cloud.githubusercontent.com/assets/3521793/17345932/13aff740-58ce-11e6-8834-623a982a028e.jpg)

**Comment by DevinWalker** [comment]
It looks like you're on the latest version of pro - Just make sure that the core has the latest version too in the `vendor/wordimpress/` subdirectory of the plugin

**Comment by lchski** [comment]
With the latest version of core, this seems to be a free and pro issue _on the front-end only_. I wonder if some of the new window code that I was working on got accidentally committed when it was generated.

**Comment by lchski** [comment]
I just put together a fix in core that I believe addresses this.

@DevinWalker, I noticed something in the code as I addressed this problem. It seems like sometimes a global `gmb.info_window` object is used to deal with the info window, and sometimes a function parameter is used. I switched to the former in this case, and it solved the problem—the function parameter hadn’t been defined, up the tree. Is this a byproduct of how the plugin JS used to be architected? Is the `gmb.info_window` global the right way to handle this, now? Just want to make sure I’m sticking with the right convention/method for the project.
